service/templates/admin.html

```diff
@@ -178,6 +178,12 @@ <h2 class="modal-title">[[modalTitle]]</h2>
                           <md-radio-button data-ng-value="false"></md-radio-button><md-radio-button data-ng-value="true"></md-radio-button>
                         </md-radio-group></td>
                       </tr>
+                      <tr>
+                        <td>Background Stats Collectors</td>
```

Comment:
Need to add github_status_api button too?

Reply:
what?